qrand.go

```diff
@@ -66,6 +67,29 @@ func Read(buf []byte) (n int, err error) {
 	return Reader.Read(buf)
 }
 
+// Source represents an alternative source that can also generate uniformly-distributed pseudo-random values
+type Source struct{}
+
+// Seed takes no argument because there is nothing to seed
+func (s *Source) Seed(seed int64) {}
+
+// Uint64 returns a pseudo-random 64-bit value as a uint64 from Source
+func (s *Source) Uint64() (value uint64) {
+	binary.Read(Reader, binary.BigEndian, &value)
+	return value
+}
+
+// Int63 returns a non-negative pseudo-random 63-bit integer as an int64 from Source
+func (s *Source) Int63() (value int64) {
+	return int64(s.Uint64() & ^uint64(1<<63))
+}
+
+// NewSource returns a pointer to qrand Source
+func NewSource() *Source {
```

Comment:
For symmetry with `math/rand`, we should return `rand.Source`, shouldn't we? In other words, we use a standard library interface rather than our own concrete type.